src/android/adb.ts

```diff
@@ -51,7 +51,9 @@ export function getApplicationPid(
 ): number {
   let output: Buffer | undefined;
   try {
-    output = execSync(`${getAbdPath(adbPath)} shell pidof -s ${applicationId}`);
+    output = execSync(
+      `"${getAdbPath(adbPath)}" shell pidof -s ${applicationId}`
```

Comment:
Shouldn't `'` be used here instead of `"`?

Reply:
Resolved